source/administration/devops-command-center.rst

```diff
@@ -1,185 +1,9 @@
-    - ``IncidentID``: Unique identifier of the incident where the item is.
-    - ``UserID``: Unique identifier of the user that edited the item."
-    "Unchecked checklist item checked", "- Any user checks an unchecked checklist item through the incident details view, in the RHS.", "
-    - ``IncidentID``: Unique identifier of the incident where the item is.
-    - ``UserID``: Unique identifier of the user that checked the item."
-    "Checked checklist item unchecked", "- Any user unchecks a checked checklist item through the incident details view, in the RHS.", "
-    - ``IncidentID``: Unique identifier of the incident where the item is.
-    - ``UserID``: Unique identifier of the user that unchecked the item."
-     "Playbook created", "- Any user clicks on the **+ New Playbook** button on the backstage and saves it.", "
-    - ``PlaybookID``: Unique identifier of the playbook.
-    - ``TeamID``: Unique identifier of the team where the playbook is created.
-    - ``NumChecklists``: Number of checklists this playbook has after the update.
-    - ``TotalChecklistItems``: Number of checklist items, among all checklists, this playbook has after the update."
-     "Playbook deleted", "- Any user clicks on the **Delete** button next to a playbook on the backstage and confirms.", "
-    - ``PlaybookID``: Unique identifier of the playbook.
-    - ``TeamID``: Unique identifier of the team where the playbook was located.
-    - ``NumChecklists``: Number of checklists this playbook had immediately prior to deletion.
-    - ``TotalChecklistItems``: Number of checklist items, among all checklists, this playbook had immediately prior to deletion."
-
-Glossary
-^^^^^^^^
-
-* **Incident:** An event requiring the coordinated actions of one or more Mattermost users. An incident is either ongoing or closed.
-* **Playbook:** A set of steps to execute as part of resolving an incident. It consists of one or more checklists, with each checklist item representing a single step.
-* **Commander:** The Mattermost user currently responsible for transitioning an incident from ongoing to closed.
-* **Incident channel:** A Mattermost channel dedicated to real-time conversation about the incident.
-* **Incident participant:** A Mattermost user with access to the corresponding incident channel.
-* **The RHS:** The incident list and incident details displayed on the right hand side of the webapp. Clicking an incident from the list in the RHS surfaces details of the selected incident. It is not available on mobile.
-* **Incident backstage:** The full-screen analytics and configuration screens accessible from the webapp. It is not available on mobile.
-* **Active incident:** An incident that has not been ended.
+The Mattermost Incident Management documentation can be found `here <https://mattermost.gitbook.io/incident-response-plugin>`_.
```

Comment:
Should we change the page URL to `.../incident-response-management`?

Reply:
The URL hasn't changed yet as it's based on the repo name, which is still ``mattermost-plugin-incident-response``. This will be changed though, just not sure when. I'll follow up on it today in the team sync.

Reply:
@justinegeffen Let's confirm in meeting today, it should be changed to `mattermost-incident-management`

Reply:
Perfect. :) The docs themselves are called Mattermost Incident Management as discussed, but the top-left corner and URL still reflect the repo name.